src/components/ucp-tabs/config-editor/ui-elements/ui-factory/CreateUIElement.tsx

```diff
@@ -128,7 +128,10 @@ function CreateUIElement(args: {
       );
     default: {
       LOGGER.msg(
-        t('gui-editor:config.element.unsupported.type', JSON.stringify(spec)),
+        localize({
```

Comment:
Ah okay, so `localize` is used together with a `useMessage` to put something in the console log (and file log).
How would I do this outside of a React Context? `getStore().get(GUI_LOCALIZATION_ATOM)` ?